user/alloctst.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
This tests the positive side of things, which is good.
But you also need to check that an invalid access results in a fault/crash.

Reply:
Okay, I will add those tests too.